gslib/tests/test_mb.py

```diff
@@ -133,3 +133,53 @@ def test_create_with_pap_invalid_arg(self):
                             expected_status=1,
                             return_stderr=True)
     self.assertRegexpMatches(stderr, r'invalid_arg is not a valid value')
+
+  @SkipForXML('Rpo flag only works for GCS JSON API')
```

Comment:
Looks like periods are missing from some of these decorator messages

Reply:
Done.